### ISSUE TITLE
 atrial septal defect, ostium primum type MONDO:0020437

### ISSUE THREAD

**Issue by kanems** [monarch-initiative/mondo#3879]
**Mondo term (ID and Label):**
MONDO:0020437  atrial septal defect, ostium primum type

**Xref that should be fixed (ID and label):**
MeSH "C548006" is obsolete (as of2017 https://id.nlm.nih.gov/mesh/C548006.html) 

**Your nano-attribution (ORCID)**
If you don't have an ORCID, you can sign up for one [here](https://orcid.org/)

**Other comments:**
This term appears to overlap/match https://hpo.jax.org/app/browse/term/HP:0010445 
Which UMLS has (unfortunately) split across two different CUIs-  C0344735 "Partial atrioventricular canal" is where most of the HPO syns/terms map (also UMLS X-refs SNOMED_CT 253373002: ' Atrioventricular septal defect - isolated atrial component' and syn 'Primum atrial septal defect')
but then the one HPO term that is a closest match to the preferred name on this Mondo term gets mapped to CUI C0031192 "Persistent Ostium Primum" (which also maps 'Ostium primum defect' from SNOMEDCT_US, PT,  17718000 and 'Atrial Septal Defect Ostium Primum' from MeSH, D006344)  So I'm not sure which (or maybe both?) would be the best match to this Mondo concept and could be added as X-refs instead of the retired MeSH code. (The CUI from Orpha:99106 is C0741296, but that was retired in 2017 and merged with C0031192, so maybe this is the best match? But perhaps UMLS is unnecessarily splitting hairs here and they are equivalent concepts, I don't know.)

**Comment by peupeubangbang** [comment]
Thanks @kanems and @nicolevasilevsky its very useful to see how this exact match failed QC (MESH:M0007390)!

This would probably be a regular issue with the broad MeSH **D** topical descriptor terms which include lots of narrow synonyms that then prevent prevent the exact and more narrow **M** concept identifier terms being used as an xref after ontology QC looks for matching synonyms from MeSH... so I might stay away from adding **M** concept IDs as xrefs for the time being.

Not a huge deal by any means as I see the value in MeSH being more the **D** terms for indexing anyway 😄
